server/views/pages/bookAVisit/visitors.njk

```diff
@@ -79,35 +80,66 @@
               text: restriction.comment,
               classes: "bapv-force-overflow test-restrictions-comment" + loop.index
             },
-            {
-              text: restrictionStartDate,
-              classes: "test-restrictions-start-date" + loop.index
-            },
             {
               text: restrictionExpiryDate,
               classes: "test-restrictions-end-date" + loop.index
             }
           ]), prisonerRestrictions) %}
         {% endfor %}
+
+        {% set prisonerAlerts = [] %}
+        {% for alert in activeAlerts %}
+          {% set alertExpiryDate %}{% if alert.dateExpires %}{{ alert.dateExpires | formatDate }}{% else %}No end date{% endif %}{% endset %}
+          {%- set prisonerAlerts = (prisonerAlerts.push([
+            {
+              html: '<span class="test-alert-type' + loop.index + ' govuk-tag alert-tag alert-tag--' + alert.alertType + '">' + alert.alertTypeDescription + '</span>'
+            },
+            {
+              html: bapvExpandableComment({ text: alert.comment }),
+              classes: "test-alert-comment" + loop.index
```

Comment:
```suggestion
              attributes: { "data-test": "alert-comment-" + loop.index }
```
I know some of the existing code on this page is using `test-` class names but I'd be tempted to move over to `data-test` attributes for consistency with how we do things now. Some existing ones where we have eg `<span class="test-xyz"...` could be swapped to use `data-test` too.